app/services/images/optimizer.rb

```diff
@@ -81,5 +103,14 @@ def self.get_imgproxy_endpoint
         ApplicationConfig["IMGPROXY_ENDPOINT"] || "http://localhost:8080"
       end
     end
+
+    def self.extract_suffix_url(full_url)
+      prefix = "https://#{ApplicationConfig['CLOUDFLARE_IMAGES_DOMAIN']}/cdn-cgi/image"
```

Comment:
nit: I assume this `extract_suffix_url` was service-agnostic, but having move `CLOUDFLARE_IMAGES_DOMAIN` here, that's no longer the case no longer the case. This isn't important right now though.